lib/msal-common/src/index.ts

```diff
@@ -93,3 +93,4 @@ export { TimeUtils } from "./utils/TimeUtils";
 export { ServerTelemetryManager } from "./telemetry/server/ServerTelemetryManager";
 export { ServerTelemetryRequest } from "./telemetry/server/ServerTelemetryRequest";
 export { PopTokenGenerator } from "./crypto/PopTokenGenerator";
+export { version } from "./packageMetadata";
```

Comment:
Do we need this?

Reply:
We might want it later. At worst it's a consistency item, this is a good thing to have available.

Reply:
Alright. Approving.